Accessability with chuanqi amin

AmineHLub opened the pull request:
<h2> Summary </h2>

- **Page titles** - Change Title to be more **precise.**
- **Image text alternatives**** (img alt="alt text") - Change Alt <img> to be more **specific**
- **Text headings** - Check on headers …
- **Color contrast** - Change light colors to have bit more contrast for **readability** …
- **Resize** - no issues found
- **Interaction** - no issues found
- **Moving content** - no issues found
- **Multimedia** - no issues found
- **The basic structure of the page** - no issues found

AmineHLub committed:
Change Title to be more precise.

AmineHLub committed:
Change Alt <img> to be more specific

AmineHLub committed:
Check on headers

change last h1 to p (paragraph)

AmineHLub committed:
Change light colors to have abit more contrast for readability

Add hover on buttons

chuanqizhu0820 committed:
Change light colors to have abit more contrast for readability

Add hover on buttons

chuanqizhu0820 committed:
"Resize, Interaction, Moving content, Multimedia, The basic structure of the page“ - no issues found"

chuanqizhu0820 committed:
Merge branch 'accessability-with-chuanqi-Amin' of https://github.com/AmineHLub/Microverse-Portfolio into accessability-with-chuanqi-Amin

chuanqizhu0820 committed:
"Resize, Interaction, Moving content, Multimedia, The basic structure of the page" - no issues found"

mke2111 reviewed (changes requested):
### Hey @AmineHLub, 

**This is a great project you have going on, well-done lad**. 

   
**However, there are a few sections to improve before your project is approved.** 🔴 
<hr>

## Required Changes ♻️
_Check the inline comments under the review for sections to improve._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

mke2111 commented on index.html:
```diff
@@ -22,7 +22,7 @@
     <link rel="stylesheet" href="./more-style/desktop-cards.css">
     <link rel="stylesheet" href="./more-style/desktop-contact.css">
-    <title>Portfolio</title>
+    <title>Portfolio | Amin</title>
```
- [ ] It is better to have a more meaningful page title. The last word is not displayed in the tab. Use Amin's Portfolio instead of just portfolio

AmineHLub replied:
Hello I read something about:  Making sure the focus of the title should be on the content on preview so we thought instead of using Amin's Portfolio where the focus on the readability of the tab title would be on "Amin" to make it focused instead of the content which is the portfolio but I'll change it to what you told me to. Thank you very much

mke2111 replied:
Nice of you to put in the extra work. 👍🏽 However, this is part of the recommendations for the project requirements. It is better to have the attention of any viewer with your name and not just the word portfolio.

AmineHLub committed:
Update Title

- Use "Amin's Portfolio" instead of "Portfolio | Amin" (the focus is on the name not the portfolio)

Georjane reviewed (approved):
## Status: PR Approved 💯 🥇 
#### Hello @AmineHLub, 

Your project is complete!  You have met all requirements for this project. It is a thumbs up for me. 👍🏽 You can go ahead and merge this PR. 
Congratulations! 🎉 👏👏

- [ ] Ohh and also don't forget to update your live link in your Readme file with a link that has all the recent changes you made to your portfolio!

_Good luck with your next project and Happy coding!_ 💻
